packages/odyssey-react/src/components/Text/styles.css

```diff
@@ -0,0 +1,245 @@
+/*!
+ * Copyright (c) 2021-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+@import '../../../../odyssey/src/scss/abstracts/functions';
+@import '../../../../odyssey/src/scss/abstracts/colors';
+@import '../../../../odyssey/src/scss/abstracts/mixins';
+@import '../../../../odyssey/src/scss/abstracts/tokens';
+
+:--root {
```

Comment:
I'm not quite sure what's going on here, are you trying to declare a custom property?

Reply:
The custom selector allows us to define its contents dynamically, which is how I'm using this CSS for both the CSS module component styles and the precompiled CSS asset that is created in my demo above. Using just `html` or `:root` would not give us the effect we're after as we want a scoped class for the component, and element selectors for the precompiled CSS asset.